components/team_members_dropdown/team_members_dropdown.tsx

```diff
@@ -89,27 +101,25 @@ export default class TeamMembersDropdown extends React.PureComponent {
         }
     }
 
-    handleDemote = (user, role, newRole) => {
+    public handleDemote = (user: UserProfile, role: string): void => {
         this.setState({
             serverError: this.state.serverError,
             showDemoteModal: true,
             user,
             role,
-            newRole,
         });
     }
 
-    handleDemoteCancel = () => {
+    public handleDemoteCancel = (): void => {
         this.setState({
             serverError: null,
             showDemoteModal: false,
             user: null,
             role: null,
-            newRole: null,
```

Comment:
Please correct me if I'm wrong. I don't see the state `newRole` being used in any way.

Reply:
Looks good to me. It doesn't seem to be used anywhere, so it must be old code.